tests/unit/Dashboard.spec.ts

```diff
@@ -60,37 +61,36 @@ describe('Dashboard - UI', () => {
     expect(vm.filedCount).toEqual(3)
   })
 
-  it('enables standalone filing buttons when there are no blocker filings in the to-do list', () => {
-    wrapper.find(TodoList).vm.$emit('has-blocker-filing', false)
+  it('enables standalone filing buttons when there are no blockers', () => {
+    expect(vm.disableChanges).toEqual(false)
+    expect(wrapper.find('#standalone-addresses-button').attributes('disabled')).toBeUndefined()
+    expect(wrapper.find('#standalone-directors-button').attributes('disabled')).toBeUndefined()
+  })
+
+  it('disables standalone filing buttons when there is a blocker filing in the todo list', () => {
+    wrapper.find(TodoList).vm.$emit('has-blocker-task', true)
 
-    expect(vm.hasBlockerFiling).toEqual(false)
-    expect(wrapper.find('#standalone-addresses-button')
-      .attributes('disabled')).toBeUndefined()
-    expect(wrapper.find('#standalone-directors-button')
-      .attributes('disabled')).toBeUndefined()
+    expect(vm.hasBlockerTask).toEqual(true)
+    expect(wrapper.find('#standalone-addresses-button').attributes('disabled')).toBe('true')
+    expect(wrapper.find('#standalone-directors-button').attributes('disabled')).toBe('true')
   })
 
-  it('disables standalone filing buttons when there is a blocker filing in the to-do list', () => {
-    wrapper.find(TodoList).vm.$emit('has-blocker-filing', true)
+  it('disables filing buttons when there is a future effective filing pending', () => {
```

Comment:
disables standalone filing buttons when there is a blocker filing in the filing history list